pkg/monitoring/vmistats/collector.go

```diff
@@ -121,40 +160,72 @@ func (vmc *vmiCountMetric) UpdateFromAnnotations(annotations map[string]string)
 	if val, ok := annotations[annotationPrefix+"flavor"]; ok {
 		vmc.Flavor = val
 	}
+
+	if val, ok := annotations[k6tv1.InstancetypeAnnotation]; ok {
+		vmc.InstanceType = other
+
+		for _, it := range instanceTypes.instanceTypes {
```

Comment:
If we need to lookup the vendor for a given instancetype then I'd prefer if this was a map of the instance type name to vendor.

Reply:
good idea, updated